frontend/src/components/resultPage/resultsReferredOut/ReferredOutTests.js

```diff
@@ -0,0 +1,861 @@
+                            handleLabNumberSearch(e);
+                          }}
+                        />
+                      )}
+                    </Field>
+                    <br />
+                    <div className="formInlineDiv">
+                      <div className="searchActionButtons">
+                        <Button
+                          type="button"
+                          onClick={handleReferredOutPatient}
+                        >
+                          <FormattedMessage
+                            id="referral.button.labSearch"
+                            defaultMessage="Search Referrals By Lab Number"
+                          />
+                        </Button>
+                      </div>
+                    </div>
+                  </Section>
+                </Column>
+              </Grid>
+              <hr />
+              <Grid fullWidth={true}>
+                <Column lg={16} md={8} sm={4}>
+                  <Section>
+                    <h5>
+                      <FormattedMessage id="referral.search" />
+                    </h5>
+                    <div className="formInlineDiv">
+                      <Field name="labNumber">
+                        {({ field }) => (
+                          <CustomLabNumberInput
+                            name={field.name}
+                            labelText={intl.formatMessage({
+                              id: "eorder.labNumber",
+                              defaultMessage: "Lab Number",
+                            })}
+                            id={field.name}
+                            className="inputText"
+                            value={values[field.name]}
+                            onChange={(e, rawValue) => {
+                              setFieldValue(field.name, rawValue);
+                              handleLabNumber(e);
+                            }}
+                          />
+                        )}
+                      </Field>
+                      <Field name="patientId">
+                        {({ field }) => (
+                          <TextInput
+                            name={field.name}
+                            value={values[field.name]}
+                            labelText={intl.formatMessage({
+                              id: "patient.id",
+                              defaultMessage: "Patient Id",
+                            })}
+                            id={field.name}
+                            className="inputText"
+                            onChange={handlePatientId}
+                          />
+                        )}
+                      </Field>
+                      <Field name="lastName">
+                        {({ field }) => (
+                          <TextInput
+                            name={field.name}
+                            labelText={intl.formatMessage({
+                              id: "patient.last.name",
```

Comment:
You were not supposed to re-write the patient component , you could re-use the existing component

Reply:
See how to use the existing PatientSearch Componenet  [here](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/reports/common/PatientStatusReport.js#L196-L199) in the PatientStatusReport